File: Web_admin/app/Model/BinhLuan.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletess;

class BinhLuan extends Model
{
	use SoftDeletes;
	protected $dates = ['deleted_at'];
	
	protected $fillable = [
		'Noidung',
		'Ngaydang',
		'taikhoan_id',
		'phim_id'
	];

	protected function phim()
	{
		$this->belongsTo('Model\Phim','phim_id');
	}

	protected function taikhoan()
	{
		$this->belongsTo('Model\TaiKhoan','taikhoan_id');
	}

	protected function phanhois()
	{
		$this->hasMany('Model\PhanHoi','binhluan_id','id');
	}
}

File: app/app/src/main/java/com/example/app/Model/BinhLuan.java
package com.example.app.Model;

public class BinhLuan {
    private int ID;
    private String NoiDung;
    private String TaiKhoan;
    private String NgayDang;

    public BinhLuan (int id, String taiKhoan, String noiDung, String ngayDang){
        this.ID = id;
        TaiKhoan = taiKhoan;
        NoiDung = noiDung;
        NgayDang = ngayDang;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public void setNoiDung(String noiDung) {
        NoiDung = noiDung;
    }

    public String getNoiDung() {
        return NoiDung;
    }

    public int getID() {
        return ID;
    }

    public String getNgayDang() {
        return NgayDang;
    }

    public String getTaiKhoan() {
        return TaiKhoan;
    }

    public void setNgayDang(String ngayDang) {
        NgayDang = ngayDang;
    }

    public void setTaiKhoan(String taiKhoan) {
        TaiKhoan = taiKhoan;
    }
}
